src/layouts/shared/page-header.tsx

```diff
@@ -10,6 +10,8 @@ export default function PageHeader() {
   const { t } = useTranslation();
   const [isDarkMode] = useDarkMode();
   const { fylkeskommune, urls } = getOrgData();
+  const displayContactPage =
+    process.env.NEXT_PUBLIC_DISPLAY_CONTACT_PAGE === 'true';
```

Comment:
The env variable should be a URL to the API that handles the contact form. If this URL is defined, we show the link (not use true/false)